roles/glance/templates/etc/glance/glance-api.conf

```diff
@@ -58,7 +58,7 @@ stores = glance.store.rbd.Store,
 rbd_store_ceph_conf = /etc/ceph/ceph.conf
 rbd_store_chunk_size = {{ glance.rbd_store_chunk_size }}
 rbd_store_user = glance
-rbd_store_pool = default
+rbd_store_pool = {{ ceph_default_pool }}
```

Comment:
What will happen is one runs ursula ..... site.yml --tags glance   ?
Will ceph_default_pool be defined?